manifests/server/redhat.pp

```diff
@@ -6,11 +6,24 @@
 #
 class nfs::server::redhat inherits nfs::client::redhat {
 
-  service{ 'nfs':
+  $servicename = $::operatingsystemmajrelease ? {
+    '7'     =>  'nfs-server',
+    default => 'nfs'
+
+  }
+
+  service{ $servicename:
     ensure    => running,
     enable    => true,
     hasstatus => true,
     require   => Package[ 'nfs-utils' ],
   }
 
+  @concat {'/etc/exports':
```

Comment:
I'm not a big fan of using virtual resources for this (I'd rather have a separate class) but it's the way it's not done for `nfs::server::debian` so far indeed, so we might want to just refactor it later.